MailCore/Utils/Realm+Extensions.swift

```diff
@@ -38,17 +38,4 @@ public extension Object {
 
         return transactionable.fetchObject(ofType: Self.self, forPrimaryKey: primaryKeyValue)
     }
-
-    /// Get an updated frozen copy for a given object
-    func refresh() -> Self {
```

Comment:
@Ambrdctr saw this as unused, since the code is `transactionable` oriented, I removed this.